apps/api/src/bootstrap.ts

```diff
@@ -88,8 +91,18 @@ export async function bootstrap(expressApp?): Promise<INestApplication> {
   app.use(extendedBodySizeRoutes, bodyParser.json({ limit: '20mb' }));
   app.use(extendedBodySizeRoutes, bodyParser.urlencoded({ limit: '20mb', extended: true }));
 
-  app.use(bodyParser.json());
-  app.use(bodyParser.urlencoded({ extended: true }));
+  const rawBodyBuffer = (req, res, buffer, encoding) => {
+    if (!req.headers['stripe-signature']) {
+      return;
+    }
+
+    if (buffer && buffer.length) {
+      req.rawBody = Buffer.from(buffer);
+    }
+  };
+
+  app.use(bodyParser.json({ verify: rawBodyBuffer }));
+  app.use(bodyParser.urlencoded({ extended: true, verify: rawBodyBuffer }));
```

Comment:
How difficult would it be (is it possible) to export a verify function from the BillingModule and import it here for use?
It's a little uncomfortable for separation of concerns to have `stripe-signature` header here in the OSS edition.